fabric-transitive-access-wideners-v1/build.gradle

```diff
@@ -124,4 +123,22 @@ ClassNode loadClass(Path path) {
 	return node
 }
 
+task validateTemplateAccessWidener {
```

Comment:
I think it would be better to check the generated file (using the `lines` variable) at the end of `generateAccessWidener` in case we make a mistake there too.